src/Http/Controllers/ResetController.php

```diff
@@ -2,21 +2,35 @@
 
 namespace Whitecube\LaravelCookieConsent\Http\Controllers;
 
-use Whitecube\LaravelCookieConsent\CookiesManager;
+use Illuminate\Support\Facades\Cookie;
 use Illuminate\Http\Request;
+use Whitecube\LaravelCookieConsent\CookiesManager;
+use Whitecube\LaravelCookieConsent\Facades\Cookies;
+
 
-class ResetController
-{
-    public function __invoke(Request $request, CookiesManager $cookies)
-    {
-        $response = ! $request->expectsJson()
+class ResetController {
+
+    public function __invoke(Request $request, CookiesManager $cookies) {
+        $response = !$request->expectsJson()
             ? redirect()->back()
             : response()->json([
                 'status' => 'ok',
                 'scripts' => $cookies->getNoticeScripts(true),
                 'notice' => $cookies->getNoticeMarkup(),
             ]);
 
+        $domain = config('cookieconsent.cookie.domain');
+
+        // delete all defined cookies
+        foreach (Cookies::getCategories() as $category) {
+            foreach ($category->getCookies() as $cookie) {
+                Cookie::queue(Cookie::forget(
+                    name: $cookie->name,
+                    domain: $domain,
+                ));
+            }
+        }
```

Comment:
```suggestion
        // delete all defined cookies
        $response = array_reduce($cookies->getCategories(), fn($response, $category) => array_reduce(
            $category->getCookies(),
            fn($response, $cookie) => $response->withoutCookie(cookie: $cookie->name, domain: $domain),
            $response
        ), $response);
```

Did not test, but this way we do not need to use Cookie queuing and the `Cookie` facade.